operate/importer-8_4/src/main/java/io/camunda/operate/zeebeimport/v8_4/processors/fni/FNITransformer.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.zeebeimport.v8_4.processors.fni;
+
+import static io.camunda.operate.zeebeimport.util.ImportUtil.tenantOrDefault;
+import static io.camunda.zeebe.protocol.record.intent.ProcessInstanceIntent.ELEMENT_ACTIVATING;
+import static io.camunda.zeebe.protocol.record.intent.ProcessInstanceIntent.ELEMENT_COMPLETED;
+import static io.camunda.zeebe.protocol.record.intent.ProcessInstanceIntent.ELEMENT_TERMINATED;
+
+import io.camunda.operate.entities.FlowNodeInstanceEntity;
+import io.camunda.operate.entities.FlowNodeState;
+import io.camunda.operate.entities.FlowNodeType;
+import io.camunda.operate.util.ConversionUtils;
+import io.camunda.operate.util.DateUtil;
+import io.camunda.operate.zeebeimport.cache.FNITreePathCacheCompositeKey;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.value.ProcessInstanceRecordValue;
+import java.time.Instant;
+import java.util.Set;
+import java.util.function.Function;
+
+/**
+ * Transfomer to transform a given Zeebe flow node instance record to a {@link
+ * FlowNodeInstanceEntity}.
+ */
+public class FNITransformer {
```

Comment:
The entire `importer-8_4` can be deleted because it is not required on `main`.

Reply:
Ah these duplications driving me nuts :D Sorry for that